src/Microsoft.TestPlatform.ObjectModel/Navigation/DiaSession.cs

```diff
@@ -47,7 +47,7 @@ public DiaSession(string binaryPath)
         /// search path.
         /// </param>
         public DiaSession(string binaryPath, string searchPath)
-            : this(binaryPath, searchPath, GetSymbolReader(binaryPath))
+            : this(binaryPath, searchPath, GetSymbolReader(binaryPath, !Path.IsPathRooted(binaryPath) ? Directory.GetCurrentDirectory() : null))
```

Comment:
Added code looks like magic. What are we trying to do? Is there a platform specific assumption? When would the path be not rooted?

May be we should add a comment.

Reply:
Why not move the `PathRooted` logic to `GetSymbolReader`?

Reply:
for all remote scenario, the path is always not rooted, the check for it could be moved to GetSymbolReader